src/index.html

```diff
@@ -7,55 +7,29 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
+
     <div class="container">
-      <div class="game-header">
-        <h1>2048</h1>
+      <div class="game_header">
+        <h1 class="game_title">2048</h1>
         <div class="controls">
           <p class="info">
-            Score: <span class="game-score">0</span>
+              Score: <span class="game_score" id="score">0</span>
+          </p>
+          <p class="info">
+            best: <span class="game_score" id="best">0</span>
           </p>
-          <button class="button start">Start</button>
+          <button class="button start" id="startbutton">New Game</button>
         </div>
       </div>
 
-      <table class="game-field">
-        <tbody>
-          <tr class="field-row">
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-          </tr>
-
-          <tr class="field-row">
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-            <td class="field-cell"></td>
-          </tr>
+      <div id="board" class="game_board">
```

Comment:
Class 'game_board' should be named according to BEM convention, i.e. 'game-board'.